packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3923,13 +3923,13 @@ export class ContainerRuntime
 		return this.blobManager.createBlob(blob, signal);
 	}
 
-	private submitIdAllocationOpIfNeeded(resubmitOutstandingRanges = false): void {
+	private submitIdAllocationOpIfNeeded(resubmitOutstandingRanges): void {
```

Comment:
can you please add a type for resubmitOutstandingRanges?

Reply:
oops, done!